packages/iTwinUI-react/stories/core/Header.stories.tsx

```diff
@@ -52,7 +52,7 @@ export default {
   },
   parameters: {
     creevey: {
-      delay: 1000,
+      delay: 2000,
```

Comment:
The `Full` header visual test is failing. Not sure why because nothing changed. But it's the only failure in the whole repo.
![image](https://user-images.githubusercontent.com/9084735/161339682-94e21c41-ce92-4dd6-ac4c-68ea37b6bd13.png)

I was hoping increasing the delay would fix it but it doesn't. Maybe we just update the image?

Reply:
would be intersting to see, whats the place failing..

Reply:
Looks like there are some more visual test failures now.
![image](https://user-images.githubusercontent.com/9084735/162478871-ecf81616-83b9-47c0-be6a-e9e0d9c98f0b.png)